ElectronClient/app/gui/TagItem.jsx

```diff
@@ -0,0 +1,23 @@
+const React = require('react');
```

Comment:
In TagItem.jsx and TagList.jsx, sometimes spaces are used for indentation, sometimes tabs. Please could you change them all to tabs?

Reply:
Hey, this is fixed. Since these are new files, VSCode did not have any prior indentation to detect, and used my defaults that is space.